.github/workflows/qa-testbench.yaml

```diff
@@ -0,0 +1,125 @@
+name: QA Testbench run
+
+on:
+  workflow_dispatch: {}
+  schedule:
+  # * is a special character in YAML so you have to quote this string
+    - cron:  '0 0 * * *'
```

Comment:
:wrench: Do we want to offset that by a couple of hours? Jenkins is already running QA at midnight, so maybe 1AM is a good time?